_data/projects/facedb.yml

```
@@ -0,0 +1,12 @@
+name: FaceDB
+desc: A face database for large-scale face recognition
+site: https://github.com/shhossain/facedb
+tags:
+- python
+- machine-learning
+- deep-learning
+- face-recognition
+- vector-database
+upforgrabs:
+  name: help wanted
+  link: https://github.com/shhossain/facedb/labels/help%20wanted
```

:art: this automation is being fussy:

```suggestion
  link: https://github.com/shhossain/FaceDB/labels/help%20wanted
```